CHANGELOG.md

```diff
@@ -28,6 +28,46 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - feat(logs)!: simplify metadata configuration [#2626]
 - feat(metrics)!: simplify metadata configuration [#2622]
 - feat(events)!: add config.merge option [#2643]
+- chore!: move parameters from `fluentd.logs.containers` to `sumologic.log.container` [#2635]
```

Comment:
**nit:** should this be `sumologic.logs.container` instead of `sumologic.log.container`? same for the rest

Reply:
Thank you! this is now corrected :)